### DISCUSSION TITLE
fail to insert cohesive element in 3D model

### DISCUSSION THREAD

**Discussion by crest621010** [neperfepx/neper#892] [Q&A] [answered]
I tried to instert cohesive element among grains in 3D model, but it failed because core dumped. Could you help me to fix it?
The codes are as follows：

neper -T -n from_morpho -id 1 -dim 3 -domain "cube(1,1,2)" \
      -morphooptiini "packing"  \
      -morpho "cube(1,1,2)"  \
      -o cube3D_04 
C="-datacellcol id -datacelltrs 0.5 -cameraangle 12 -imagesize 600:800"
neper -V cube3D_04.tess $C -print cube3D_04

neper -M cube3D_04.tess \
      -elttype tet \
      -interface cohesive \
      -rcl 4 \
      -o cube3D_01

**Comment by Abaqus-Voronoi-toolbox** [comment]
I think you Neper version is too lower. 
These commands are no problem in my laptop.

**Comment by Abaqus-Voronoi-toolbox** [comment]
See attached files
[Results.zip](https://github.com/user-attachments/files/16531805/Results.zip)

**Comment by rquey** [comment, marked as answer]
This discussion has been closed as obsolete. Please repost if necessary.
